docs/styleguide.config.js

```diff
@@ -59,6 +59,15 @@ module.exports = {
         '../react/Snackbar',
         '../react/Stepper',
         '../react/Switch',
+        '../react/MuiTable',
+        '../react/TableBody',
+        '../react/TableCell',
+        '../react/TableContainer',
+        '../react/TableFooter',
+        '../react/TableHead',
+        '../react/TablePagination',
+        '../react/TableRow',
+        '../react/TableSortLabel',
```

Comment:
Maybe we should call all these files starting with Mui  ('../react/MuiTableBody' instead of '../react/TableBody' )like you did with the first one '../react/MuiTable' ?

Reply:
Non à mon avis il faut plutôt laisser ça comme ça et renommer MuiTable en Table

Reply:
Check